skyportal/models.py

```diff
@@ -189,6 +190,123 @@ def desi_dr8_url(self):
                             primaryjoin='group_users.c.user_id == users.c.id')
 
 
+class Candidate(Base):
```

Comment:
Could we use a mixin with Source to simplify this one?

https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/mixins.html